.circleci/config.yml

```diff
@@ -28,47 +28,22 @@ jobs:
   Test:
     executor:
       name: ios/default
-      xcode-version: "12.0"
+      xcode-version: "12.4.0"
     steps:
       - setup_environment
       - run:
           name: Build & Test
-          # We've seen some builds timeout because CircleCI didn't receive
-          # output for 10m. Making it wait for 20m instead should address that.
-          #
-          # For example,
-          # https://app.circleci.com/pipelines/github/Automattic/simplenote-macos/1879/workflows/23bba8c2-3bc0-447e-877c-b3eadd326c12/jobs/1834
-          # timed out but its re-run
-          # https://app.circleci.com/pipelines/github/Automattic/simplenote-macos/1879/workflows/d1d5b6d6-90b0-42c4-81ff-466625e26a6b/jobs/1835
-          # succeeded
-          no_output_timeout: 20m
-          command: |
-            # Build without code signing to avoid missing cert errors
-            xcodebuild clean test \
-                -workspace 'Simplenote.xcworkspace' \
-                -scheme 'Simplenote' \
-                -configuration 'Debug' \
-                COMPILER_INDEX_STORE_ENABLE=NO \
-                CODE_SIGN_IDENTITY="" \
-                CODE_SIGNING_REQUIRED=NO
+          command: bundle exec fastlane test
```

Comment:
Love how less code there's in the CircleCI layer. Would/Will make it simpler to migrate to a different provides.

Reply:
![nod](http://static.skaip.org/img/emoticons/180x180/f6fcff/nod.gif)